downhill/base.py

```diff
@@ -103,7 +104,10 @@ def __init__(self, loss, params=None, inputs=None, updates=(), monitors=(),
             self._monitor_exprs.append(monitor)
         if monitor_gradients:
             for p, g in zip(self._params, TT.grad(self._loss, self._params)):
-                self._monitor_names.append('∂{}'.format(p.name))
```

Comment:
This is a good idea, but I'd prefer something shorter as an implementation:

    d = '∂' if sys.stdout.encoding == 'UTF-8' else 'd'
    self._monitor_names.append('{}{}'.format(d, p.name))